app/Nova/Actions/ExportUsersBuzzCardAccess.php

```diff
@@ -38,41 +31,42 @@ class ExportUsersBuzzCardAccess extends Action
     /**
      * Perform the action on the given models.
      *
-     * @param  \Laravel\Nova\Fields\ActionFields  $fields
-     * @param  \Illuminate\Support\Collection  $models
-     * @return mixed
+     * @param \Laravel\Nova\Fields\ActionFields  $fields
+     * @param \Illuminate\Support\Collection  $models
+     *
+     * @return array<string,string>
      */
-    public function handle(ActionFields $fields, Collection $models)
+    public function handle(ActionFields $fields, Collection $models): array
     {
         $population = $fields->population;
-        if ($population === 'core') {
-            $users = User::select('gtid')->BuzzCardAccessEligible()
-                ->whereHas('teams', function (Builder $query) {
-                    $query->where('name', 'Core');
-                })->get();
-        } elseif ($population === 'general') {
-            $users = User::select('gtid')->BuzzCardAccessEligible()
-                ->whereDoesntHave('teams', function (Builder $query) {
-                    $query->where('name', 'Core');
-                })->get();
-        } else {
-            return Action::danger('Invalid population!');
-        }
+        $users = User::select('gtid')->BuzzCardAccessEligible()
+            ->when(
+                'core' === $population,
+                static function (Builder $q): void {
+                    $q->whereHas('teams', static function (Builder $query): void {
+                        $query->where('name', 'Core');
+                    });
+                },
+                static function (Builder $q): void {
+                    $q->whereDoesntHave('teams', static function (Builder $query): void {
+                        $query->where('name', 'Core');
+                    });
+                }
+            )
+            ->get();
 
-        if (count($users) == 0) {
-            return Action::danger('No users match the provided criteria');
+        if (0 === count($users)) {
+            return Action::danger('No users match the provided criteria!');
         }
 
         // Exclude fields that we don't care about (Everything except GTID)
-        $users->forget("name","full_name","preferred_first_name","is_active","is_access_active");
+        $users->pluck('gtid');
 
-        $filename = "robojackets-$population-buzzcard-" . time() . '.csv';
-        $path = "nova-exports/$filename";
+        $filename = 'robojackets-'.$population.'-buzzcard-'.time().'.csv';
```

Comment:
Instead of `core` vs `general`, name the file to match what BuzzCard wants. Hopefully that will help avoid mistakes when information is lost in communication.

Reply:
Done, ex. `575F-GRP_SCC_with_RoboJackets-2021-02-13T14_10_08.csv`, `575F-GRP_SCC_without_RoboJackets-2021-02-13T14_06_03.csv`